pkg/framework/operators/supported.go

```diff
@@ -4,10 +4,12 @@ type OperatorType int
 
 const (
 	OperatorTypeQdr OperatorType = iota
+	OperatorTypeBase OperatorType = iota
 )
 
 var (
 	SupportedOperators = map[OperatorType]OperatorSetupBuilder{
 		OperatorTypeQdr: &QdrOperatorBuilder{},
+		OperatorTypeBase: &BaseOperatorBuilder{},
```

Comment:
This is just an abstract implementation right? The supported map is meant for concrete ones only.

Reply:
True. Would remove.